themes/_core/js/product.js

```diff
@@ -311,4 +311,17 @@ $(document).ready(() => {
             }
         });
     });
+
+    prestashop.on('updatedProduct', (args) => {
+        if (!args.product_url) {
+            return;
+        }
+        window.history.pushState(
+            {
+              id_product_attribute: args.id_product_attribute
+            },
+            undefined,
```

Comment:
Shouldn't this be an empty string?
https://developer.mozilla.org/en-US/docs/Web/API/History_API#The_pushState()_method

Reply:
I just took what was on the previous prestashop version and from doc:`a title (which is currently ignored), `

Reply:
I changed it to use `document.title` like in `admin-theme.js`